src/crimson/os/alienstore/alien_store.cc

```diff
@@ -380,4 +380,119 @@ unsigned AlienStore::get_max_attr_name_length() const
   return 256;
 }
 
+seastar::future<struct stat> AlienStore::stat(
+  CollectionRef ch,
+  const ghobject_t& oid)
+{
+  using stat_struct = struct stat;
+  return seastar::do_with(stat_struct(), [=] (auto& st) {
```

Comment:
i'd suggest be explicit when capturing local variables. see also https://google.github.io/styleguide/cppguide.html#Lambda_expressions